modules/Miss.js

```diff
@@ -40,12 +40,18 @@ class Miss extends React.Component {
     return (
       <LocationSubscriber>
         {(location) => {
-          const { render, component:Component } = this.props
+          const { children, render, component:Component } = this.props
           const { noMatchesInContext } = this.state
           const { serverRouter, match } = this.context
           const noMatchesOnServerContext = serverRouter &&
             serverRouter.missedAtIndex(match.serverRouterIndex)
-          if (noMatchesInContext || noMatchesOnServerContext) {
+          const matched = !(noMatchesInContext || noMatchesOnServerContext)
```

Comment:
So, when there is no match in context, it means it matches the `Miss` component and its children know that the parent (`Miss`) is matched. Got it. Thank you @spicydonuts. Will fix it then.